examples/service-clients/azure-client/external-controller/src/fluid.ts

```diff
@@ -0,0 +1,110 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+	AzureLocalConnectionConfig,
+	AzureRemoteConnectionConfig,
+} from "@fluidframework/azure-client";
+// eslint-disable-next-line import/no-internal-modules -- #26985: `test-runtime-utils` internal used in example
+import { InsecureTokenProvider } from "@fluidframework/test-runtime-utils/internal";
+import {
+	type ContainerSchema,
+	type IFluidContainer,
+	SharedTree,
+	TreeViewConfiguration,
+} from "fluid-framework";
+import { v4 as uuid } from "uuid";
+
+import { AzureFunctionTokenProvider } from "./AzureFunctionTokenProvider.js";
+import { TwoDiceApp, Dice } from "./schema.js";
+
+export interface ICustomUserDetails {
+	gender: string;
+	email: string;
+}
+
+const userDetails: ICustomUserDetails = {
+	gender: "female",
+	email: "xyz@microsoft.com",
+};
+
+// Define the server we will be using and initialize Fluid
+const useAzure = process.env.FLUID_CLIENT === "azure";
+
+const user = {
+	id: uuid(),
+	name: uuid(),
+};
+
+const azureUser = {
+	id: user.id,
+	name: user.name,
+	additionalDetails: userDetails,
+};
+
+export const connectionConfig: AzureRemoteConnectionConfig | AzureLocalConnectionConfig =
+	useAzure
+		? {
+				type: "remote",
+				tenantId: "",
+				tokenProvider: new AzureFunctionTokenProvider("", azureUser),
+				endpoint: "",
+			}
+		: {
+				type: "local",
+				tokenProvider: new InsecureTokenProvider("fooBar", user),
+				endpoint: "http://localhost:7070",
+			};
+
+/**
+ * Schema for the Dice Roller Container.
+ * This includes the DataObjects we support and any initial DataObjects we want created
+ * when the Container is first created.
+ */
+export const diceRollerContainerSchema = {
```

Comment:
Organizationally, wondering if the tree and schema-specific stuff might make more sense in controller.ts and/or schema.ts?  No strong opinion though.